src/stories/containers/FinancesOverview/FinancesOverviewContainer.tsx

```diff
@@ -54,84 +55,82 @@ const FinancesOverviewContainer: React.FC<FinancesOverviewContainerProps> = ({
-          <ExpensesChartColumn>
-            <ExpensesChart newActual={newActual} newDiscontinued={newDiscontinued} newPrediction={newPrediction} />
-            {!isDownTable && <NavigationButtons />}
-          </ExpensesChartColumn>
+          <BreakdownSectionContainer>
+            <ExpensesChartColumn>
+              <ExpensesChart newActual={newActual} newDiscontinued={newDiscontinued} newPrediction={newPrediction} />
+              {!isDownTable && <NavigationButtons />}
+            </ExpensesChartColumn>
 
-          <BreakdownTableColumn>
-            <CostBreakdownTable
-              selectedFilter={selectedFilter}
-              setSelectedFilter={setSelectedFilter}
-              byBudgetExpenses={byBudgetExpenses}
-              remainingBudgetCU={remainingBudgetCU}
-              remainingBudgetDelegates={remainingBudgetDelegates}
-              maxValueByBudget={maxValueByBudget}
-              byCategoryExpenses={byCategoryExpenses}
-              remainingCategories={remainingCategories}
-              maxValueByCategory={maxValueByCategory}
-              total={costBreakdownTotal}
-              handleOnClick={handleOpenModal}
-            />
-          </BreakdownTableColumn>
-          {isDownTable && <NavigationButtons />}
-        </BreakdownSectionContainer>
-      </Container>
-      <CategoryModalComponent
-        checkOut={checkOut}
-        headCountCategories={headCountCategory}
-        notHeadCountCategory={notHeadCountCategory}
-        handleCloseModal={handleCloseModal}
-        handleCheckedExpandedAll={handleCheckedExpandedAll}
-        handleChangeItemAccordion={handleChangeItemAccordion}
-        isLight={isLight}
-        openModal={openModal}
-      />
+            <BreakdownTableColumn>
+              <CostBreakdownTable
+                selectedFilter={selectedFilter}
+                setSelectedFilter={setSelectedFilter}
+                byBudgetExpenses={byBudgetExpenses}
+                remainingBudgetCU={remainingBudgetCU}
+                remainingBudgetDelegates={remainingBudgetDelegates}
+                maxValueByBudget={maxValueByBudget}
+                byCategoryExpenses={byCategoryExpenses}
+                remainingCategories={remainingCategories}
+                maxValueByCategory={maxValueByCategory}
+                total={costBreakdownTotal}
+                handleOnClick={handleOpenModal}
+              />
+            </BreakdownTableColumn>
+            {isDownTable && <NavigationButtons />}
+          </BreakdownSectionContainer>
+        </Container>
+        <CategoryModalComponent
+          checkOut={checkOut}
+          headCountCategories={headCountCategories}
+          notHeadCountCategory={noHeadCountCategories}
+          handleCloseModal={handleCloseModal}
+          handleCheckedExpandedAll={handleCheckedExpandedAll}
+          handleChangeItemAccordion={handleChangeItemAccordion}
+          isLight={isLight}
+          openModal={openModal}
```

Comment:
Except, maybe the `openModal` prop, all the other props could be accessed through the hook inside the modal

Reply:
you are right, all are except the handleOpenModal